test/helper.sh

```diff
@@ -1,7 +1,20 @@
 #!/usr/bin/env bash
 
+set -o errexit
+set -o nounset
+set -o pipefail
+
 ROOT="$(dirname "${BASH_SOURCE[0]}")/.."
 
+# check whether command is installed.
+function cmd_must_exist {
+    local CMD=$(command -v ${1})
+    if [[ ! -x ${CMD} ]]; then
+      echo "error: Please install ${1} and verify they are in \$PATH."
+      exit 1
+    fi
+}
+
```

Comment:
```suggestion
# check whether command is installed.
function cmd_must_exist {
    if ! command -v ${1} > /dev/null 2>&1; then
      echo "error: Please install ${1} and verify they are in \$PATH."
      exit 1
    fi
}

```

For builtin and alias commands, use -x will judgment fails. 
We could just use `command -v`.